portfolio/src/main/webapp/script.js

```diff
@@ -18,32 +18,106 @@
 function getCommentSection(){
   fetch('/load-comments').then(response => response.json()).then((comments) => {
```

Comment:
since you are getting comments here, I mean why not just count comments here , something like 
```
totalComments.innerText = comments.length
```

Reply:
The comments variable that gets passed in the response is already trimmed down to only the amount of comments the user wants to see on the screen, so comments.length would return the amount of comments to display, not the total amount of comments in the datastore. I wasn't sure how to pass the total amount of comments in the datastore without a dummy variable. Am I thinking about this incorrectly? I recognize that the dummy variable is probably a brute force method and not very good practice but it seems that the size of the data store is something I can only access in the servlet.

Reply:
I see your points now. Optionally, my 2 cents:
1. you could have one API to get total comments
2. you could have a data structure to return both comments and total number of comments.Something like
```commentsMetadata = {total: totalComments, displayComments: displayComments}```

But you may make this as improve later if you have spare time.